### PR TITLE
Feat: Add admin, employee and store schemas to db.

### PR DESCRIPTION
**Pull request by GustavoHiguera**
Ps: didn't add relations

Co-authored-by: dgentner <dgentner@ucol.mx>
Co-authored-by: CalixtoLopez <calixto.lopez.santos@gmail.com>
Co-authored-by: JorgeRodz <jarf4321@gmail.com>

# Description
What did you implemented/Why did you implemented this:
We migrated the database with "rails db:migrate", also we missed the email on employee, so we did the migration again.

### CODE REVIEW

**Review by elioflo — APPROVED** [review]
Good Job!

File: time_checker/db/migrate/20220901213331_create_admins.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class CreateAdmins < ActiveRecord::Migration[6.1]
+  def change
+    create_table :admins do |t|
+      t.string :first_name
+      t.string :last_name
+      t.string :email
+      t.string :password
```
Comment (willcab) [inline time_checker/db/migrate/20220901213331_create_admins.rb:7]: the password should be encrypted below [you can use this as a guide](https://www.bigbinary.com/blog/rails-7-adds-encryption-to-active-record)

**Review by willcab — APPROVED** [review]
look good to me

**Review by iMartinDav — APPROVED** [review]
Look good

**Pull request merged**